pallets/oracle/Cargo.toml

```diff
@@ -5,46 +5,45 @@ authors = ["Pendulum"]
 edition = "2021"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "max-encoded-len"] }
-scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
+codec = { workspace = true, features = ["derive", "max-encoded-len"] }
+scale-info = {  workspace = true, features = ["derive"] }
 
 # Substrate dependencies
-sp-std = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
-sp-arithmetic = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false, optional = true }
+sp-std = { workspace = true }
```

Comment:
Shouldn't we put `default-features = false` again here?

Reply:
Connected to this [one](https://github.com/pendulum-chain/spacewalk/pull/569#discussion_r1858226124), so basically since that is overridden by the workspace definition, we go the other way around, set `default_features = false` on the workspace, and then if needed, we override with `default_features = true`, which works.